alejandroidepaz/DataTrack
# Dev/tests and styling

What I did:

- Fixed styling of chart icons on home page
- Added support for `npm test`
- Added test for Home component

To test:
- Build the app and make sure the chart icons on the homepage render in an evenly-spaced grid
- run `npm test` (may also have to run `npm install` first)
